mitxgraders/formulagrader/matrixgrader.py

```diff
@@ -59,3 +80,31 @@ def check_response(self, answer, student_input, **kwargs):
             else:
                 return {'ok': False, 'msg': err.message, 'grade_decimal': 0}
         return result
+
+    def default_equality_comparer(self, comparer_params, student_input, utils):
+        """
+        Default comparer function.
+
+        Assumes comparer_params is just the single expected answer wrapped in a list.
+        """
+        try:
+            return utils.within_tolerance(comparer_params[0], student_input)
+        except ShapeError as err:
+            is_raised = self.config['answer_shape_mismatch']['is_raised']
+            msg_detail = self.config['answer_shape_mismatch']['msg_detail']
+
+            if msg_detail is None:
+                msg = ''
+            else:
+                detailed = msg_detail == 'shape'
+                expected_shape = get_description(comparer_params[0], detailed=detailed)
+                received_shape = get_description(student_input, detailed=detailed)
+                if not detailed and expected_shape == received_shape:
+                    msg = ("Exepected answer to be a {0}, but input is a {1}."
+                           "of incorrect shape.".format(expected_shape, received_shape))
```

Comment:
Wait, did this just run on from the period directly?

Reply:
Do you mean this?
```python
continued_string = ( "a"
                                   "b" )
continued_string == "ab" # True
```

But the period was a typing. Looking at this more closely, apparently I forgot `assert` in all of my tests! (except the error-raising ones). So that needs to be fixed.

Reply:
First line: "... but input is a {1}." (ends in a period).
Second line: "of incorrect shape."

I think you want to change the period to a space?

Reply:
Yep, and it didn't break the tests because I forgot the asserts. =/